databroker/headersource/base.py

```diff
@@ -1,8 +1,16 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 from . import core
+import numpy as np
 
 
+def _sanitize_np(val):
```

Comment:
It's possible that this is actually safe to remove once #195 is in, which does handle it lower at least in places. Let's revisit there. If you don't want to delete the underscore, that's fine, just trying to save 5 seconds of hassle while I have just burned here. :- )